recipes/iqtree/meta.yaml

```diff
@@ -1,40 +1,43 @@
 {% set name = "IQTREE" %}
 {% set version = "2.3.5" %}
-{% set sha256 = "8e323e0b7c46e97901d3500f11e810703e0e5d25848188047eca9602d03fa6b1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version | replace("-", "_") }}
 
 build:
-  number: 1
+  number: 2
   run_exports:
     - {{ pin_subpackage('iqtree', max_pin="x") }}
 
 source:
-  - url: https://github.com/iqtree/iqtree2/archive/refs/tags/v{{ version }}.tar.gz
-    sha256: {{ sha256 }}
-  - url: https://github.com/tothuhien/lsd2/archive/refs/tags/v.2.4.1.tar.gz
-    sha256: 3d0921c96edb8f30498dc8a27878a76d785516043fbede4a72eefd84b5955458
-    folder: lsd2
+  git_url: https://github.com/iqtree/iqtree2.git
+  git_rev: v{{ version }}
```

Comment:
Is this the same as https://github.com/iqtree/iqtree2/archive/refs/tags/v2.3.5.tar.gz ?

Reply:
If it is not the same then maybe we should use something like https://github.com/bioconda/bioconda-recipes/pull/49453/files#diff-36db4cc0b96656522ba2e09be03ea10d6b9d2f9d648926887db6a9706dd8acdaR10-R11. This way you can keep using the `sha256` check

Reply:
No it's not. This repo has submodules and these are only automatically included if using git URL instead of tarball. Hence the switch (there are two submodules: lsd2 and cmaple. It's much cleaner to use the git checkout than to manually have to keep the subrepo commits up to date by hand. In fact, they were previously outdated. 

See https://github.com/bioconda/bioconda-recipes/issues/5375#issuecomment-321617390